spec/services/resident_list_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+require_relative '../../app/services/resident_list'
+
+include AuthenticationSpecHelpers
```

Comment:
This include is in the wrong spot, this needs to be inside the RSpec.describe block if you are going to use it.
However, we don't even need this module here. This is only if we need to test the API and need to include a JWT in the headers for an authenticated user.

Reply:
Gotcha! 🫡